### PR TITLE
Fix name of nodes ckecked in test_annotate

### PR DESCRIPTION
**Pull request by asi1024**
Part of #193

### CODE REVIEW

**Comment by kmaehashi** [comment]
/test

Let's see if these tests pass with PyTorch 1.8 or 1.7. If fails I think it's ok to skip and just support the latest PyTorch in ppe.onnx.

@take-cheeze It would be great if you could take a look on diff whether this fix is reasonable :pray:

File: tests/pytorch_pfn_extras_tests/onnx/test_annotate.py [hunk 1]
```diff
@@ -141,6 +155,7 @@ def _fn2(x):
     assert 'yyy' in node_attrs
 
 
+@pytest.mark.filterwarnings("ignore::DeprecationWarning")
```
Comment (asi1024) [inline tests/pytorch_pfn_extras_tests/onnx/test_annotate.py:158]: This DeprecationWarning seems to be raised only on the first call of `export_testcase`.

**Comment by take-cheeze** [comment]
memo: CI failure of warning would be fixed by https://github.com/pfnet/pytorch-pfn-extras/pull/197

**Comment by asi1024** [comment]
@take-cheeze @kmaehashi 
Fixed to use `Linear(..., bias=False)` to simplify computation graph.

**Review by take-cheeze — APPROVED** [review]
LGTM 👍

**Comment by kmaehashi** [comment]
LGTM, thanks @asi1024 for fixing and @take-cheeze for reviewing!

**Pull request merged**